project-roles/trusted-committer.md

```diff
@@ -163,5 +163,5 @@ Published internally at Nike; drafted via pull-request in June of 2018.
 [Loren Sanz]: https://github.com/mrsanz
 [Jeremy Hicks]: https://github.com/greatestusername
 [Noah Cawley]: https://github.com/utanapishtim
-[praise]: https://github.com/paypal/InnerSourcePatterns/blob/master/praise-participants.md
+[praise]: https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/praise-participants.md
```

Comment:
I'm not sure how to make that a relative link...

Reply:
figured it out thanks to [github docs](https://github.github.com/gfm/#reference-link)
(TIL)